src/QueryStrategies/BatchQueryStrategies/EnumerativeBatchQs.jl

```diff
@@ -0,0 +1,66 @@
+mutable struct EnumerativeBatchQs <: MultiObjectiveBatchQs
+    model::SVDD.OCClassifier
+    inf_measure::SequentialPQs
+    rep_measure::Function
+    div_measure::Function
+    normalization::Function
+    k::Int
+    λ_inf::Float64
+    λ_rep::Float64
+    λ_div::Float64
+
+    function EnumerativeBatchQs(model::SVDD.OCClassifier, informativeness::SequentialPQs; representativeness::Symbol=nothing, diversity::Symbol=nothing,
+        k::Int=0, λ_inf::T1 where T1<:Real =0.33, λ_rep::T2 where T2 <: Real=0.33, λ_div::T3 where T3 <: Real=0.33)::EnumerativeBatchQs
+        # check basic params
+        (model == nothing) && throw(ArgumentError("No model specified."))
+        (k < 1) && throw(ArgumentError("Invalid batch size k=$(k)."))
+
+        # min max normalization
+        function normalization(x::Vector{Float64})::Vector{Float64}
+            return (x .- min(x...)) ./ (max(x...) - min(x...))
+        end
```

Comment:
This function is repeated in other batch query strategies. Please extract this into a helper.